packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -91,24 +88,30 @@ export class MetamaskPage implements WalletPage {
   async switchNetwork(networkName = 'Linea Mainnet') {
     await test.step(`Switch network to "${networkName}"`, async () => {
       await this.navigate();
-      await this.header.networkSetting.switchNetwork(networkName);
+      await this.header.networkList.switchNetwork(networkName);
       await this.page.close();
     });
   }
 
   async setupNetwork(standConfig: Record<string, any>) {
     await test.step(`Setup "${standConfig.chainName}" Network`, async () => {
-      await this.header.networkSetting.networkListButton.click();
+      const currentNetwork = await this.header.getCurrentNetworkName();
+      if (currentNetwork.includes(standConfig.chainName)) {
```

Comment:
For fork this check will be incorrect. 
Because for fork we should to use same chain name (chainId) and then to add additional rpc url only

Reply:
For correclty check i created isNetworkExist method

Reply:
you can check errors here https://github.com/lidofinance/lido-autotests/actions/runs/11569905748/job/32204577885

Reply:
Do I need to remove 98-101 lines to fix that?

Reply:
removed, done